experimenter/experimenter/targeting/constants.py

```diff
@@ -1668,7 +1668,6 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
-
```

Comment:
Removed some extra newlines between a few nearby constants to make spacing in the file more uniform 🤷

Reply:
Oh huh that should usually be handled by autoformatting....